mobile/src/main/java/rs/readahead/washington/mobile/views/activity/onboarding/OnBoardingActivity.kt

```diff
@@ -87,13 +86,14 @@ class OnBoardingActivity : BaseActivity(), OnBoardActivityInterface,
     }
 
     private fun initUwaziEvents() {
-        createServer.observe(this,
-            { server: UWaziUploadServer? ->
-                if (server != null) {
-                    presenter.create(server)
-                    addFragment(OnBoardHideOptionFragment(),R.id.rootOnboard)
-                }
-            })
```

Comment:
I would suggest adding the back buttons inside the `OnBordingActivity` layout file, instead of adding them in each Fragment.